flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/parser/JaninoCompiler.java

```diff
@@ -300,7 +315,19 @@ private static Java.Rvalue generateOtherFunctionOperation(
         }
     }
 
-    private static Java.Rvalue generateNoOperandTimestampFunctionOperation(String operationName) {
+    private static Java.Rvalue generateTimezoneFreeTemporalFunctionOperation(String operationName) {
+        List<Java.Rvalue> timestampFunctionParam = new ArrayList<>();
+        timestampFunctionParam.add(
+                new Java.AmbiguousName(Location.NOWHERE, new String[] {DEFAULT_EPOCH_TIME}));
+        return new Java.MethodInvocation(
+                Location.NOWHERE,
+                null,
+                StringUtils.convertToCamelCase(operationName),
+                timestampFunctionParam.toArray(new Java.Rvalue[0]));
```

Comment:
I wonder if is it necessary to initialize `timestampFunctionParam` here, since it seems did the same thing:

```java
new Java.Rvalue[] {
    new Java.AmbiguousName(Location.NOWHERE, new String[] {
        DEFAULT_EPOCH_TIME
    })
}
```